gammapy/datasets/make.py

```diff
@@ -64,3 +71,121 @@ def sigma_energy_theta(energy, theta, sigma):
                                        zenith=zenith_hi)
 
     return psf
+
+
+def make_test_observation_table(observatory, n_obs):
+    """Generate an observation table.
+
+    For the moment, only random observation tables are created.
+
+    Parameters
+    ----------
+    observatory : string
+    	name of the observatory
+    n_obs : integer
+    	number of observations for the obs table
+
+    Returns
+    -------
+    obs_table : `~gammapy.obs.ObservationTable`
+    	observation table
+    """
+    n_obs_start = 1
+
+    obs_table = ObservationTable()
+
+    # obs id
+    obs_id = np.arange(n_obs_start, n_obs_start + n_obs)
+    col_obs_id = Column(name='OBS_ID', data=obs_id)
+    obs_table.add_column(col_obs_id)
+
+    # on time
+    ontime = Quantity(30.*np.ones_like(col_obs_id.data), 'minute')
+    col_ontime = Column(name='ONTIME', data=ontime)
+    obs_table.add_column(col_ontime)
+
+    # livetime
+    livetime = Quantity(25.*np.ones_like(col_obs_id.data), 'minute')
+    col_livetime = Column(name='LIVETIME', data=livetime)
+    obs_table.add_column(col_livetime)
+
+    # TODO: add columns for coordinates:
+    #       alt az
+    #       date -> done
+    #       calculate pointing observation (ra dec), considering the observatory location, and alt az is at the middle of the observation
+    #       then convert to alt az, considering the observatory location
+
+    # TODO: is there a way to comment on the column names?!!!!!
+    # TODO: store obs name as a column or as a header value?!!!
+    # TODO: format times!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+    # TODO: alt az at mean time!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+    # TODO: restrict to night time? (or dark time?)!!!!!!!!!!!!
```

Comment:
I'd say don't bother making these fake tables more realistic, leave as is and move on to the observation planning toolbox. Once `astroplan` is in place I plan to do https://github.com/gammapy/gammapy/issues/79#issue-27908311 as a use case and then it will get much more realistic. For now the purpose is only to test the observation handling toolbox you implement, and for this you don't need a good fake.